### DISCUSSION TITLE
本地搭建Doris FE的开发环境为什么会遇到 gpg: 签名时失败

### DISCUSSION THREAD

**Discussion by longzmkm** [apache/doris#33097] [A - General / Q&A]
![image](https://github.com/apache/doris/assets/19727358/40de0bac-5a4a-4c4c-9660-3c50df4295b3)
捣鼓了大半天了， 哪位大佬能够告诉我这个解决办法

**Comment by yagagagaga** [comment]
It looks like the Internet is bad and the downloaded files are incomplete.
